lib/rspec/rails/file_fixture_upload_support.rb

```diff
@@ -1,16 +1,16 @@
 module RSpec
   module Rails
     # @private
-    module FixtureFileUploadSupport
-      delegate :fixture_file_upload, to: :rails_fixture_file_wrapper
+    module FileFixtureUploadSupport
+      delegate :file_fixture_upload, :fixture_file_upload, to: :rails_file_fixture_wrapper
```

Comment:
One minor note... I have included the delegation for `file_fixture_upload` in *all* Rails versions, even though it's only available in Rails 7.1.+. It should raise a `NoMethodError` if called in Rails < 7.1.

Reply:
I'm confused as to what provides the old behaviour with this change, I can see the build passing but I don't understand what now provides `fixture_file_upload`

Reply:
@JonRowe it's hard to see, since `file_fixture_upload` and `fixture_file_upload` look so similar, but `fixture_file_upload` is on this same `delegate` line, after `file_fixture_upload`.

Reply:
Ah I see, sorry!

Reply:
No problem!